rosdep/python.yaml

```diff
@@ -8003,6 +8011,10 @@ python3-scipy:
       depends: [gfortran]
       packages: [scipy]
   ubuntu: [python3-scipy]
+python3-scipy-pip:
+  ubuntu:
+    pip:
+      packages: [scipy]
```

Comment:
```suggestion
```

Please see the conversation in https://github.com/ros/rosdistro/pull/25615 for why we are not adding pip keys for packages which are available from distribution package managers.